source/testing/index.md

```diff
@@ -1,79 +1,149 @@
 Testing is a core part of the Ember framework and its development cycle.
 
 Let's assume you are writing an Ember application which will serve as a blog.
-This application would likely include models such as `user` and `post`. It would
-also include interactions such as _login_ and _create post_. Let's finally
-assume that you would like to have [automated tests] in place for your application.
+This application would likely include models such as `user` and `post`.
+It would also include interactions such as _login_ and _create post_.
+Let's finally assume that you would like to have [automated tests] in place for your application.
 
-There are three different classifications of tests that you will need:
-**Acceptance**, **Unit**, and **Integration**.
+There are four different types of test setups you can choose from depending on what you will need to test.
+Let's have a look at those test types and when each of them is the most useful.
 
```

Comment:
I think it's good to list all 4 types in this intro. It may even be worthwhile to mention that "in the past, these were sometimes referred to as Acceptance, Unit, and Integration tests." I could see this change making it hard for people to look back at old materials and figure out what's going on.